sympy/printing/llvmjitcode.py

```diff
@@ -136,6 +136,74 @@ def _print_Symbol(self, s):
         return value
 
 
+class LoopCreator(object):
```

Comment:
The LoopCreator class is supposed to be internal to this file.  I'll add a leading underscore.

Yes, you're right, there's substantially enough here now that I should add it to the documentation.